Stratis.Bitcoin/Features/Wallet/Wallet.cs

```diff
@@ -107,6 +107,23 @@ public IEnumerable<Script> GetAllPubKeysByCoinType(CoinType coinType)
                 yield return address.ScriptPubKey;
             }
         }
+
+        /// <summary>
+        /// Adds an account to the current wallet.
+        /// </summary>
+        /// <remarks>The name given to the account is of the form "account (i)" by default, where (i) is an incremental index starting at 0.
```

Comment:
you mean like this:
<remarks>
The name given to the account is of the form "account (i)" by default, where (i) is an incremental index starting at 0.
</remarks>